app/views/users/show.html.erb

```diff
@@ -1,16 +1,38 @@
 <h1>User Page with List of Posts</h1>
-<p>User 1</p>
-<p>Bio: Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+<div class="user-line">
+    <%= link_to image_tag(@user.photo, class: 'user-photo'), user_path(@user) %>
+    <div class="name-counter">
+      <span class="name">
+        <%= link_to @user.name, user_path(@user) %>
+      </span>
+      <div>
+        <span class="right-top"></span>
+        <span class="counter">Number of posts: <%= @user.posts_counter %></span>
+        <span class="right"></span>
+      </div>
+    </div>
+</div>
+<section>
+  <h4>Bio:</h4>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
   sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
   Ac tortor dignissim convallis aenean et. Lectus magna fringilla urna 
   porttitor rhoncus dolor. Consequat interdum varius sit amet mattis 
-  vulputate. Fringilla urna porttitor rhoncus dolor purus non enim. 
-  Pharetra massa massa ultricies mi quis hendrerit. Praesent elementum 
-  facilisis leo vel. Ultrices neque ornare aenean euismod elementum 
-  nisi quis eleifend quam. Turpis cursus in hac habitasse platea 
-  dictumst quisque sagittis.</p>
-<ul>
-  <li>Post 1</li>
-  <li>Post 2</li>
-  <li>Post 3</li>
-</ul>
+  vulputate. Fringilla urna porttitor rhoncus dolor purus non enim.</p>
+</section>
+
+<% @posts.each do |post| %>
+  <div class="post-border">
+    <p>Post <%= post.id %>: <%= post.title %></p>
+  </div>
+<% end %>
```

Comment:
- [ ] 🌟 Great job on your progress! However, there's a refinement needed in the `users/1` route. Currently, you're displaying all posts for a user, but according to the wireframe, you should showcase only the 3 most recent posts per user. Fortunately, you've already implemented a method in your model, `three_most_recent_posts`, to retrieve these posts. I recommend utilizing this method instead of displaying all posts. This adjustment will enhance the efficiency and clarity of your code.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/jcmunav63/my-ror-blog/assets/55840999/a1b6a6c0-27ed-478a-8587-736cc77442bc)| ![image](https://github.com/jcmunav63/my-ror-blog/assets/55840999/760e6437-7a35-43f3-ab8b-262a2d9aeb86)|

Keep up the excellent work, and feel free to reach out if you have any questions or need further assistance! 🚀👩‍💻